src/main/java/com/kaleido/fetch/service/FetchService.java

```diff
@@ -152,19 +152,19 @@ public FetchService(KaptureClient <Media> mediaKaptureClient, KaptureClient <Bat
 
     public List <String> getAllBarcodes(String activityName) {
         List <String> barcodeArray = new ArrayList <String> ();
-
         ResponseEntity <List<Experiment>> experimentList = experimentKaptureClient.findByFieldEquals("name", activityName);
-        var expertimentId = experimentList.getBody().get(0).getId().toString();
-        if (experimentList.getStatusCode().is2xxSuccessful() && experimentList.getBody() != null && experimentList.getBody().size() > 0) {
-            var platmapsfromrepository = plateMapKaptureClient.findByFieldEquals("experimentId", expertimentId);
-            if (platmapsfromrepository.getStatusCode().is2xxSuccessful() && platmapsfromrepository.getBody() != null) {
-                platmapsfromrepository.getBody().stream()
-                    .forEach(plate -> {
-                        barcodeArray.add(plate.getBarcode());
-                    });
-            }
+        if(null!=experimentList.getBody() && !experimentList.getBody().isEmpty()) {
+        	var expertimentId = experimentList.getBody().get(0).getId().toString();
+        	if (experimentList.getStatusCode().is2xxSuccessful() && experimentList.getBody() != null && experimentList.getBody().size() > 0) {
+        		var platmapsfromrepository = plateMapKaptureClient.findByFieldEquals("experimentId", expertimentId);
+        		if (platmapsfromrepository.getStatusCode().is2xxSuccessful() && platmapsfromrepository.getBody() != null) {
```

Comment:
Please do a find and replace of tabs with 4 spaces

Reply:
Replaced tab with spaces